packages/aws-cdk-lib/aws-s3/lib/bucket.ts

```diff
@@ -1400,6 +1400,145 @@ export abstract class TargetObjectKeyFormat {
   public abstract _render(): CfnBucket.LoggingConfigurationProperty['targetObjectKeyFormat'];
 }
 
+/**
+ * Specifies which Amazon S3 objects to replicate and where to store the replicas.
+ */
+export interface ReplicationRule {
+  /**
+   * A container for information about the replication destination.
```

Comment:
can we also document here that currently destination can be same account or cross acount

Reply:
The addition to the text is not a problem. I just want to confirm the intention: are there any other destinations for replication besides same-account or cross-account?
For example, if we want to set up replication to both the same account and a cross-account simultaneously, we can do this by configuring multiple ReplicationRules.

Reply:
i meant `currently` to keep the scope for any additional use cases that we might have under this, if not then same account or cross account should be okay.